plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/GitExtension.java

```diff
@@ -154,5 +159,18 @@ public GitExtension(GitResources resources,
         compareGroup.add(compareWithLatestAction);
         actionManager.registerAction("gitCompareWithBranch", compareWithBranchAction);
         compareGroup.add(compareWithBranchAction);
+        actionManager.registerAction("gitCompareWithRevision", compareWithRevisionAction);
+        compareGroup.add(compareWithRevisionAction);
+
+        DefaultActionGroup gitCompareContextMenuGroup = new DefaultActionGroup("Git", true, actionManager);
+        actionManager.registerAction("GitCompareContextMenu", gitCompareContextMenuGroup);
```

Comment:
another actions have names that start with small letter but this action has name that starts with big letter. Why?